common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -42,12 +47,10 @@
   private static final String MOJO_PIPELINE_PATH_PROPERTY = "mojo.path";
```

Comment:
nit pick: i prefer to prefix properties with a context prefix (e.g., for mojos we are using `ai.h2o.mojo`)

Reply:
@mmalohlava - this is an existing property that we already set up in runtime, changing this might affect some customers who are already consuming it